utils/src/http/hyper_helpers.rs

```diff
@@ -99,7 +99,7 @@ mod tests {
 		let expected: jsonrpc::Request = serde_json::from_str(s).unwrap();
 		let body = hyper::Body::from(s.to_owned());
 		let headers = hyper::header::HeaderMap::new();
-		let bytes = read_response_to_body(&headers, body, HttpConfig::default()).await.unwrap();
+		let bytes = read_response_to_body(&headers, body, 10 * 1024 * 1024).await.unwrap();
```

Comment:
Maybe make 10 * 1024 * 1024 a const for consistency; i've seen it a few times but making a const would make it easier to change the buffer_max_size in the future if we choose to do so, and also makes it a bit more readable

I think it's readable as it is, but would be nice QoL, not sure if it's actually used enough to be worth it though

Reply:
I agree but I don't want address it in this PR already large enough :(

Filed #257